src/argilla/server/commons/telemetry.py

```diff
@@ -33,51 +38,34 @@
     Client = None
 
 
-def _configure_analytics(disable_send: bool = False) -> Client:
-    API_KEY = settings.telemetry_key or "C6FkcaoCbt78rACAgvyBxGBcMB3dM3nn"
-    TELEMETRY_HOST = "https://api.segment.io"
-
-    # Check host connection
-    httpx.options(TELEMETRY_HOST, timeout=1, verify=False)
-
-    return Client(
-        write_key=API_KEY,
-        gzip=True,
-        host=TELEMETRY_HOST,
-        send=not disable_send,
-        max_retries=5,
-    )
+_LOGGER = logging.getLogger(__name__)
 
 
 @dataclasses.dataclass
-class _TelemetryClient:
-    client: Client
+class TelemetryClient:
+    enable_telemetry: dataclasses.InitVar[bool] = settings.enable_telemetry
+    disable_send: dataclasses.InitVar[bool] = False
+    api_key: dataclasses.InitVar[str] = settings.telemetry_key or "C6FkcaoCbt78rACAgvyBxGBcMB3dM3nn"
```

Comment:
Maybe we should move this string key to the `settings` class? If you move it to the settings I think it would be nice to have it as a constant at `argilla._constants` and using that constant on the settings.

Reply:
DONE